portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,66 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body onload="background()">
+    <div id="intro-content" style="display:block;">
+      <div id="intro-button-one" style="display:inline-block">
+        <div class="mdc-touch-target-wrapper" style="display:block;">
+          <button class="mdc-button mdc-button--outlined" onclick="intro()" style="color:white;">  
+	        <span class="mdc-button__ripple">Intro</span> 
+	      </button>
+        </div>
+        <div class="mdc-touch-target-wrapper" style="display:block;">
+	      <button class="mdc-button mdc-button--outlined" onclick="tech()" style="color:white;">  
+	        <span class="mdc-button__ripple">Tech Skills</span> 
+	      </button>
+        </div>
+        <div class="mdc-touch-target-wrapper" style="display:block;">
+	      <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
+	        <span class="mdc-button__ripple">Interests</span>
+	      </button>
+        </div>
+      </div>
+      <div id="basic-profile" style="display:inline-block;">
+        <img src="/images/PROFILE_PIC.JPG">
+      	<h1>Michaela Murray's Portfolio</h1>
+      	<div id="intro">
+      	  <ul style="list-style-type:none;">
+            <li>Stanford Class of 2022</li> 
+            <li>Major: Computer Science</li>
+            <li>STEP Intern @ Google</li>
+	      </ul>
+        </div>
```

Comment:
This indentation still looks off in the review tool: 
![image](https://user-images.githubusercontent.com/2066940/85600339-3abee400-b602-11ea-865b-e0b1cc28ee63.png)
In particular the list elements are indented less than the parent `ul`.

Reply:
do you have any recommendations for fixing this? I can't see it on my end :(

Reply:
Okay I edited it directly in github...hopefully that works lol

Reply:
Interesting - the indentation still looks off in the diffing tool:
![image](https://user-images.githubusercontent.com/2066940/85606684-41505a00-b608-11ea-82dd-997d80ebd8ef.png)

But let's not block on this for this review. I've asked the intern team for input on what normalization tools they'd like us to use. We can follow up offline later to resolve this for the team.